docs/s3.md

```diff
@@ -0,0 +1,70 @@
+# SLV S3 bucket
+
+**NOTE: This document is only relevant to puppet employees**
+
+The SLV team has an s3 bucket for storing results from performance testing
+
+`
+s3://slv-performance-results
+`
+
+## Convention
+
+S3 doesn't have true directories.  But it does like to make it look like it does.  The convention 
```

Comment:
This is a side effect of object storage. You can link to the following for an explanation of this behavior.
https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html